extras/mmu.py

```diff
@@ -4419,10 +4425,15 @@ def cmd_MMU_CHECK_GATES(self, gcmd):
             gates_tools = []
             if tools != "!":
                 # Tools used in print (may be empty list)
+                if tools == 'AUTOMATIC':
```

Comment:
I want to be really clear that I'm not dead-set on anything here. I wanted to get it running enough to test, but I have no strong feelings about how this macro should interface with `_get_toolchanges_from_file`